Remotion/Web/Development.WebTesting.IntegrationTests/ScreenshotTest.cs

```diff
@@ -661,6 +667,27 @@ public void ScreenshotTransformationTest ()
       Helper.RunScreenshotTestExact<ScreenshotTest> (PrepareTest(), ScreenshotTestingType.Browser, test);
     }
 
+    private void RetryTest (Action action, int retries)
+    {
+      for (int i = 0; i < retries; i++)
+      {
+        try
+        {
+          action();
+        }
+        catch (AssertionException) when (i + 1 == retries)

```

Comment:
can we change this to use the when only to trigger the continue and not include the catch/throw-block at all? The way it's now does work but it seems overly complex.